### DISCUSSION TITLE
How do I create missing albums?

### DISCUSSION THREAD

**Discussion by DanGarion** [krateng/maloja#287] [General]
Now that I've been using this for a while I wanted to do some data cleanup. I have some songs that I know that the data is correct from the source but they aren't going to their album. In addition, the album doesn't exist at all. How can I create the album under an artist so I can then associate the songs to the album?

**Comment by duckcantcode** [comment]
I think you're going to have to do a manual scrobble and put the album name in, then merge your tracks to that album, then delete that manual scrobble. Unless there's a way to create albums that I don't know of.

**Reply by DanGarion** [reply]
That is what I thought I might have to do. I will try doing it that way.
